tc-health-client/build/tc-health-client.logrotate

```diff
@@ -16,7 +16,7 @@
 # specific language governing permissions and limitations
 # under the License.
 #
-/var/log/trafficcontrol-cache-config/tm-health-client.log {
+/var/log/trafficcontrol-cache-config/tc-health-client.log {
```

Comment:
The log is now located at `/var/log/trafficcontrol/tc-health-client.log`

Reply:
@zrhoffman changed